### PR TITLE
Member review updates

### PR DESCRIPTION
**Pull request by ved-rivos**
Summary:

1. Replace platform devices with list of initiators excluded from IOM_020.

### CODE REVIEW

File: src/server_soc_requirements.adoc [hunk 1]
```diff
@@ -1266,6 +1266,24 @@ data centers and enterprises.
...
+             Reference Model section and Ecosystem Security Objectives section
+             of the RISC-V Security Model cite:[SEC]. The Server SoC is
+             classified as a complex security system for the purposes of
+             SR_ROT_001 and SR_ATT_002.
+
+| SEC_006 a| The Server SoC MUST support the Generic System Without Supervisor
+             Domains use case detailed in the RISC-V Security Model. The
+             building blocks used to implement this use case MUST comply with
+             the requirements specified in the RISC-V Security Building Blocks
+             section of the RISC-V Security Model specification.
```
Comment (rsahita) [inline src/server_soc_requirements.adoc:1279]: should say something about section 4.2 related requirements - SR_GEN_007..016 are included I presume

**Review by rsahita — APPROVED** [review]
lgtm - one more comment - what about crypto requirements (section 5 of sec model)

**Review by NicholasWoodIMG — APPROVED** [review]

**Comment by ved-rivos** [comment]
> lgtm - one more comment - what about crypto requirements (section 5 of sec model)

Included the cryptography section in SEC_005

File: src/server_soc_requirements.adoc [hunk 2]
```diff
@@ -96,13 +96,18 @@ deliver external interrupts to the RISC-V application processor harts.
...
-             Access block.
+             The following initiators are exempt from this requirement:
+
+             * Interrupt controllers, such as the APLIC.
+             * IOMMUs.
+             * System Bus Access blocks of Debug Modules.
+             * Controllers, including the root of trust (RoT) controller, power
+               management controllers, or other SoC management controllers, when
+               they access resources reserved for their use.
+
```
Comment (ved-rivos) [inline src/server_soc_requirements.adoc:110]: Add a note that other similar initiators may be excluded from the list.

File: src/server_soc_requirements.adoc [hunk 3]
```diff
@@ -1266,6 +1266,24 @@ data centers and enterprises.
 [%header, cols="5,25"]
 |===
 | ID#     ^| Requirement
+| SEC_005 a| The Server SoC MUST comply with the requirements and guidelines
+             detailed in Reference Model, Ecosystem Security Objectives, and
+             the Cryptography sections of the RISC-V Security Model cite:[SEC].
```
Comment (ved-rivos) [inline src/server_soc_requirements.adoc:1271]: Ensure that the citation has the version included.

Comment (ved-rivos) [reply src/server_soc_requirements.adoc:1271]: Add version number to the normative requirement.

File: src/server_soc_requirements.adoc (hunk 2 above)
Comment (ved-rivos) [reply src/server_soc_requirements.adoc:110]: Text updated.

File: src/server_soc_requirements.adoc (hunk 3 above)
Comment (ved-rivos) [reply src/server_soc_requirements.adoc]: Text updated.

**Pull request merged**